docs/concepts/resource-manager.md

```diff
@@ -109,6 +109,7 @@ The gardener-resource-manager can manage a resource in different modes. The supp
 - `Ignore`
     - The corresponding resource is removed from the ManagedResource status (`.status.resources`). No action is performed on the cluster - the resource is no longer "managed" (updated or deleted).
     - The primary use case is a migration of a resource from one ManagedResource to another one.
+    - If this annotation is present on the ManagedResource itself then it will be skipped entirely by the controller (no reconciliations or deletions at all). This can be helpful to temporarily patch/change resources managed as part of such ManagedResource. 
```

Comment:
Question - 
What will be the behavior of shoot deletion? Can It be the case that If the MR is annotated with `Ignore` annotation and because of it shoot can get stuck in the deletion phase cause it can't delete all MRs?

Reply:
Yes, good question, this is what would happen. We can decide whether we keep it this way given that this feature should only be used for temporary operations purposes, or whether we want to still delete resources even if they are annotated with the "ignore mode".
I'm tending towards the second approach, although it might be a bit confusing. Opinions?

Reply:
I personally would prefer the second one i.e. `delete resources even if they are annotated with the "ignore mode"` because when the user wishes to delete the cluster he/she prioritizes deletion also cluster/shoot deletion gives the sense that I am getting rid of everything related to that shoot.
If we choose the first option It will make life more complex and have a bad user experience. It's my perspective we can take more opinion on this.